cmd/apiserver/main.go

```diff
@@ -3,16 +3,18 @@ package main
 import (
 	goflag "flag"
 	"fmt"
-	"log"
 	"os"
 
-	"github.com/golang/glog"
+	kitLog "github.com/go-kit/kit/log"
+	"github.com/go-stack/stack"
+
```

Comment:
ocd alarm.

I'm trying to only have 3 groups in the imports in the following order:

`stdlib`
[empty line]
`external deps`
[empty line]
`kubernikus internal deps`